test/interpreter/function-ceiling.spec.ts

```diff
@@ -1,95 +1,59 @@
-      ['=CEILING(1, 2, "baz")'],
-    ])
-
-    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
-    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
-    expect(engine.getCellValue(adr('A3'))).toEqualError(detailedError(ErrorType.VALUE, ErrorMessage.NumberCoercion))
-  })
-
-  it('should ensure that value and significance have same sign', () => {
-    const engine = HyperFormula.buildFromArray([
-      ['=CEILING(1, -5)'],
-      ['=CEILING(-2, 4)'],
-    ])
-
-    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NUM, ErrorMessage.DistinctSigns))
-    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.NUM, ErrorMessage.DistinctSigns))
-  })
-
-  it('should work', () => {
-    const engine = HyperFormula.buildFromArray([
-      ['=CEILING(4.43, 0.3)'],
-      ['=CEILING(4.43, 0.6)'],
-      ['=CEILING(4.43, 2)'],
-      ['=CEILING(4.43)'],
-      ['=CEILING(-4.43)'],
-      ['=CEILING(-3.14, -1.8)'],
-      ['=CEILING(-3.14, 0)'],
-      ['=CEILING(3.14, 0)'],
-    ])
-
-    expect(engine.getCellValue(adr('A1'))).toEqual(4.5)
-    expect(engine.getCellValue(adr('A2'))).toEqual(4.8)
-    expect(engine.getCellValue(adr('A3'))).toEqual(6)
-    expect(engine.getCellValue(adr('A4'))).toEqual(5)
-    expect(engine.getCellValue(adr('A5'))).toEqual(-4)
-    expect(engine.getCellValue(adr('A6'))).toEqual(-1.8)
-    expect(engine.getCellValue(adr('A7'))).toEqual(0)
-    expect(engine.getCellValue(adr('A8'))).toEqual(0)
-  })
-
-  it('should work with mode for negative numbers', () => {
-    const engine = HyperFormula.buildFromArray([
-      ['=CEILING(-11, -2)'],
-      ['=CEILING(-11, -2, 0)'],
-      ['=CEILING(-11, -2, 1)'],
-      ['=CEILING(-11, 0, 1)'],
-      ['=CEILING(-11, 0, 0)'],
-    ])
-
-    expect(engine.getCellValue(adr('A1'))).toEqual(-10)
-    expect(engine.getCellValue(adr('A2'))).toEqual(-10)
-    expect(engine.getCellValue(adr('A3'))).toEqual(-12)
-    expect(engine.getCellValue(adr('A4'))).toEqual(0)
-    expect(engine.getCellValue(adr('A5'))).toEqual(0)
-  })
-
-  it('mode should make no difference for positive arguments', () => {
-    const engine = HyperFormula.buildFromArray([
-      ['=CEILING(11, 2, 0)'],
-      ['=CEILING(11, 2, 1)'],
-    ])
-
-    expect(engine.getCellValue(adr('A1'))).toEqual(12)
-    expect(engine.getCellValue(adr('A2'))).toEqual(12)
-  })
-
-  /* Inconsisitency with Product 1*/
-  it('should work as described in ODDF', () => {
-    const engine = HyperFormula.buildFromArray([
-      ['=CEILING(-11, -2)'],
-    ])
-
-    expect(engine.getCellValue(adr('A1'))).toEqual(-10)
-  })
-})
+import {ErrorType, HyperFormula} from '../../src'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function CEILING', () => {
+  /*Inconsistent with ODDF standard.*/
```

Comment:
Interesting, so the ODDF compilant is the CEILING.MATH in Excel?

Reply:
yes